app/templates/buyers/dashboard.html

```diff
@@ -30,15 +30,7 @@
                             <div class="banner-success-without-action">
                         {% endif %}
                                 <p class="banner-message">
-                        {% if 'requirements_deleted' in message %}
-                                Your requirements &lsquo;{{ message.get('requirements_deleted') }}&rsquo; were deleted
-                        {% elif 'requirements_withdrawn' in message %}
-                                You've withdrawn your requirements for &lsquo;{{ message.get('requirements_withdrawn') }}&rsquo;.
-                        {% elif 'updated-brief' in message %}
-                                You've updated '{{ message.get('updated-brief') }}'
-                        {% else %}
-                            {{ message }}
-                        {% endif %}
+                                    {{ message }}
```

Comment:
Is it worth taking this time to refactor the few lines around this to follow the structure of `brief_overview.html` template so it's also using the toolkit macro rather than raw HTML?

At a glance it seems possible and I assume wouldn't change any behaviour but worth double checking

Reply:
Yeah I had a go and it was going to cause some layout issues (too much vertical whitespace seems to be an easy thing to end up with).

So I decided it made more sense to do all of that in one go once the template gets centralized.

Reply:
Captured here: https://trello.com/c/8Ix9Q5vn/371-flash-messages-centralize-the-template-all-apps